config-docker.toml

```diff
@@ -0,0 +1,46 @@
+[pulse]
+userid = "guest"
+host = "pulse"
+port = 5672
+exchange = "exchange/git-hg-sync/test"
+routing_key = "git-hg-sync"
+queue = "queue/git-hg-sync/sync"
+password = "guest"
+ssl = false
+
+[sentry]
+sentry_url = ""
+
+[clones]
+directory = "/clones"
+
+[[tracked_repositories]]
+name = "firefox-releases"
+url = "/clones/test-repo-git"
+
+# BETA
+[[branch_mappings]]
+source_url = "/clones/test-repo-git"
+branch_pattern = "beta"
+destination_url = "/clones/mozilla-beta" # also, m-c
```

Comment:
We do have a dev `conduit-testing/m-c` repo, so would be good to be more explicit about this here. But not sure what this comment means?

Reply:
Hum, I think this was a note-to-self more than anything else. It evolved into #24.

The docker-local test setup uses http://github.com/mozilla-conduit/test-repo as a base, and create all the test repos, so it's not really M-C. My reasoning was that it would be overkill [most of the time] to bandy around multiple copies of M-C.